source/elements/oneDAL/source/introduction.rst

```diff
@@ -2,19 +2,19 @@
 Introduction
 ============
 
-oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
```

Comment:
When I look into 
https://github.com/napetrov/oneapi-spec/blob/dev/napetrovintroduction/source/elements/oneDAL/source/introduction.rst#id1 I see 
**|dal_short_name|** is a library that provi

Reply:
yes. i've replace all cases across document with |dal_short_name|